docs/visual-basic/misc/the-folder-cannot-be-created-since-a-file-already-exists-with-the-same-path.md

```diff
@@ -15,6 +15,6 @@ An attempt has been made to create a directory, but a file already exists with t
 ## See also
 - [How to: Create a Directory](../../visual-basic/developing-apps/programming/drives-directories-files/how-to-create-a-directory.md)
 - [My.Computer.FileSystem](xref:Microsoft.VisualBasic.FileIO.FileSystem)
- [My.Computer.FileSystem.CreateDirectory]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.CreateDirectory)  
- [My.Computer.FileSystem.RenameDirectory]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)  
+- [My.Computer.FileSystem.CreateDirectory](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.CreateDirectory)
+- [My.Computer.FileSystem.RenameDirectory](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)
```

Comment:
The xref should be xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory%2A